chia/data_layer/data_layer.py

```diff
@@ -135,14 +134,5 @@ async def get_root(self, store_id: bytes32) -> Optional[bytes32]:
         res = await self.data_store.get_tree_root(tree_id=store_id)
         if res is None:
             self.log.error(f"Failed to get root for {store_id.hex()}")
+            return None
```

Comment:
This is pointless but the whole roots/`None` path needs reviewed so we don't need to burn CI cycles I guess right now.